animealUI/Assemblers/TabBarControllerAssembler.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import UIKit
+import UIComponents
+import Style
+
+class TabBarControllerAssembler {
```

Comment:
Do we need `class` here? We could get by with `struct` or `enum`

Reply:
changed to enum